apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/Chat.tsx

```diff
@@ -175,10 +174,16 @@ export default function Chat({
         content: [{ type: ContentType.text, text: input }],
       }
 
-      setResponseStream('')
-
       addMessageToConversation(message)
 
+      setError(undefined)
+      setResponseStream('')
+      setIsStreaming(true)
+      setUsage({
+        promptTokens: 0,
+        completionTokens: 0,
+        totalTokens: 0,
+      })
```

Comment:
Do a function called `resetStreaming` or something like this to make more clear all these things go together